### ISSUE TITLE
shed-tools options -y and -t are confusing

### ISSUE THREAD

**Issue by Petraea** [galaxyproject/ephemeris#72]
Hi there,
I just had a devil of a time getting shed-tools to work because of confusion between passing in a yml tool file (-t) and passing in a yml tool string (-y). Is there some way to clarify this  further in the documentation?

**Comment by Petraea** [comment]
A comment from the Oslo 2018 Elixir training course. :)

**Comment by rhpvorderman** [comment]
Sure. Can you exactly state how you were confused?

**Comment by drosofff** [comment]
-f instead of -t ? 

I have to admit that -t is a bit counter intuitive for a file, plus it reminds the -t option of planemo commands which stands for toolshed targets

**Comment by rhpvorderman** [comment]
I fail to see how `-t` for `tools file` can be extremely confusing. But I indeed see how the `-y` for the yaml string can be confusing if there also is a yaml tool file is quite confusing.
Maybe we should include an example in the usage for the `-y` flag?

**Comment by drosofff** [comment]
I didn't say it's confusing ;-) But IMHO in `tools file` the most important word is `file`, especially in the perspective of running a command line.

**Comment by rhpvorderman** [comment]
I agree, but everyone who has used this tool before now does expect a `-t` flag. And I feel bad about breaking their assumptions ;-).

**Comment by drosofff** [comment]
I understand quite well ! I would be among the impacted users ;-)
What about -t OR -f mapping to the same arg ? Not sure it is in the best practices list tough

**Comment by rhpvorderman** [comment]
Since we already broke everyone's script with release 0.8 I see no objection in doing this again. But I propose then that we use the same flags for all ephemeris tools. Now `run-data-managers` uses `--config` for its file, for example. Let's unify this so ephemeris becomes more coherent.
Are there some UNIX conventions that we can use? Hate to reinvent the wheel here ;-).
